app/models/concerns/licenses/scopes.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Licenses::Scopes
+  extend ActiveSupport::Concern
+
+  included do
+    scope :past_due, -> { where "#{table_name}.paid_until < :today", today: Time.zone.today }
```

Comment:
Casi que mejor así en estos casos de un solo parámetro:
```suggestion
    scope :past_due, -> { where "#{table_name}.paid_until < ?", Time.zone.today }
```